### DISCUSSION TITLE
JT-1: DTAB Surfactant CGenFF

### DISCUSSION THREAD

**Discussion by Jacqueline-teixeira** [Global-Chem/global-chem#198] [General]
Hi,

Could you tell me if this dihedral is foreseen in the potential of CGenFF? The head of this surfactant is a choline and the tail is a hydrocarbon. 
![vmdscene](https://user-images.githubusercontent.com/89286741/181935619-9d059ad1-322e-49fb-87bc-6c7b046cc611.png)

 362  321  318  318 NOT FOUND
 NG3P0   CG324  CG321 CG321

 270  321  318  318 NOT FOUND
 HGP5 CG324  CG321 CG321

**Comment by Sulstice** [comment]
huh I will take a look on monday. @asukaorr Do you wanna run through your versions and I'll had the SMILES version one.

**Reply by Jacqueline-teixeira** [reply]
Thanks Sulstice, 
Here this coordinate file.

51 
 
NG3P0          6.10700        0.11200       -0.00500  
CG324          4.79600       -0.60400        0.07400  
CG321          3.53800        0.29200       -0.02600  
CG321         2.26200       -0.55500        0.06700   
CG321          1.00800        0.32200       -0.03100  
CG321          -0.26600       -0.52600        0.06300 
CG321         -1.52000        0.35100       -0.03500  
CG334         6.26400        0.81400       -1.30100   
CG334         6.26300        1.08200        1.10500   
CG334         7.18600       -0.90000        0.10800   
CG321         -2.79400       -0.49700        0.05900  
CG321         -4.04800        0.38000       -0.03800  
CG321         -5.32200       -0.46800        0.05600  
CG321         -6.57500        0.40900       -0.04100  
CG321         -7.84800       -0.44000        0.05300  
CG331         -9.09900        0.43000       -0.04300  
HGP5          4.73600       -1.16100        1.03800   
HGP5          4.73700       -1.36000       -0.74300   
HGA2          3.52400        0.83000       -0.99700   
HGA2          3.52300        1.03000        0.80200   
HGA2          2.25200       -1.09900        1.03800   
HGA2          2.25300       -1.29900       -0.75900   
HGA2          1.01500        0.86600       -1.00100   
HGA2          1.01400        1.06600        0.79600   
HGA2         -0.27400       -1.27000       -0.76400   
HGA2         -0.27400       -1.07000        1.03400   
HGA2         -1.51300        1.09500        0.79200   
HGA2         -1.51200        0.89500       -1.00500   
HGP5          7.27800        1.26600       -1.39000   
HGP5          5.53900        1.64800       -1.40800   
HGP5          6.12500        0.11400       -2.15500   
HGP5          5.53800        1.91900        1.02500   
HGP5          7.27700        1.54200        1.09400   
HGP5          6.12300        0.58700        2.09200   
HGP5          7.12300       -1.44900        1.07500   
HGP5          7.12400       -1.64800       -0.71400   
HGP5          8.19300       -0.42700        0.05700   
HGA2         -2.80100       -1.04100        1.03000   
HGA2         -2.80100       -1.24100       -0.76800   
HGA2         -4.04000        1.12400        0.78900   
HGA2         -4.04000        0.92400       -1.00900   
HGA2         -5.32900       -1.21200       -0.77100   
HGA2         -5.32900       -1.01200        1.02700   
HGA2         -6.56800        1.15300        0.78600   
HGA2         -6.56800        0.95300       -1.01200   
HGA2         -7.86400       -1.18300       -0.77300   
HGA2         -7.86400       -0.98300        1.02300   
HGA3         -9.12000        1.16900        0.78500   
HGA3        -10.00400       -0.21000        0.02800   
HGA3         -9.12000        0.96900       -1.01500   
BRGA1         9.99764        0.75575       -0.21806

**Comment by asukaorr** [comment]
Sure, can you provide the mol2 file?

**Reply by Jacqueline-teixeira** [reply]
Hi @asukaorr , thanks for reply. 

51 0 0 0 0
SMALL
GASTEIGER

@<TRIPOS>ATOM
      1 *           6.1070    0.1120   -0.0050 NG3P0   1  UNL1        0.0000
      2 *           4.7960   -0.6040    0.0740 CG324   2  UNL2        0.0000
      3 *           3.5380    0.2920   -0.0260 CG321   3  UNL3        0.0000
      4 *           2.2620   -0.5550    0.0670 CG321   4  UNL4        0.0000
      5 *           1.0080    0.3220   -0.0310 CG321   5  UNL5        0.0000
      6 *          -0.2660   -0.5260    0.0630 CG321   6  UNL6        0.0000
      7 *          -1.5200    0.3510   -0.0350 CG321   7  UNL7        0.0000
      8 *           6.2640    0.8140   -1.3010 CG334   8  UNL8        0.0000
      9 *           6.2630    1.0820    1.1050 CG334   9  UNL9        0.0000
     10 *           7.1860   -0.9000    0.1080 CG334  10  UNL10       0.0000
     11 *          -2.7940   -0.4970    0.0590 CG321  11  UNL11       0.0000
     12 *          -4.0480    0.3800   -0.0380 CG321  12  UNL12       0.0000
     13 *          -5.3220   -0.4680    0.0560 CG321  13  UNL13       0.0000
     14 *          -6.5750    0.4090   -0.0410 CG321  14  UNL14       0.0000
     15 *          -7.8480   -0.4400    0.0530 CG321  15  UNL15       0.0000
     16 *          -9.0990    0.4300   -0.0430 CG331  16  UNL16       0.0000
     17 *           4.7360   -1.1610    1.0380 HGP5   17  UNL17       0.0000
     18 *           4.7370   -1.3600   -0.7430 HGP5   18  UNL18       0.0000
     19 *           3.5240    0.8300   -0.9970 HGA2   19  UNL19       0.0000
     20 *           3.5230    1.0300    0.8020 HGA2   20  UNL20       0.0000
     21 *           2.2520   -1.0990    1.0380 HGA2   21  UNL21       0.0000
     22 *           2.2530   -1.2990   -0.7590 HGA2   22  UNL22       0.0000
     23 *           1.0150    0.8660   -1.0010 HGA2   23  UNL23       0.0000
     24 *           1.0140    1.0660    0.7960 HGA2   24  UNL24       0.0000
     25 *          -0.2740   -1.2700   -0.7640 HGA2   25  UNL25       0.0000
     26 *          -0.2740   -1.0700    1.0340 HGA2   26  UNL26       0.0000
     27 *          -1.5130    1.0950    0.7920 HGA2   27  UNL27       0.0000
     28 *          -1.5120    0.8950   -1.0050 HGA2   28  UNL28       0.0000
     29 *           7.2780    1.2660   -1.3900 HGP5   29  UNL29       0.0000
     30 *           5.5390    1.6480   -1.4080 HGP5   30  UNL30       0.0000
     31 *           6.1250    0.1140   -2.1550 HGP5   31  UNL31       0.0000
     32 *           5.5380    1.9190    1.0250 HGP5   32  UNL32       0.0000
     33 *           7.2770    1.5420    1.0940 HGP5   33  UNL33       0.0000
     34 *           6.1230    0.5870    2.0920 HGP5   34  UNL34       0.0000
     35 *           7.1230   -1.4490    1.0750 HGP5   35  UNL35       0.0000
     36 *           7.1240   -1.6480   -0.7140 HGP5   36  UNL36       0.0000
     37 *           8.1930   -0.4270    0.0570 HGP5   37  UNL37       0.0000
     38 *          -2.8010   -1.0410    1.0300 HGA2   38  UNL38       0.0000
     39 *          -2.8010   -1.2410   -0.7680 HGA2   39  UNL39       0.0000
     40 *          -4.0400    1.1240    0.7890 HGA2   40  UNL40       0.0000
     41 *          -4.0400    0.9240   -1.0090 HGA2   41  UNL41       0.0000
     42 *          -5.3290   -1.2120   -0.7710 HGA2   42  UNL42       0.0000
     43 *          -5.3290   -1.0120    1.0270 HGA2   43  UNL43       0.0000
     44 *          -6.5680    1.1530    0.7860 HGA2   44  UNL44       0.0000
     45 *          -6.5680    0.9530   -1.0120 HGA2   45  UNL45       0.0000
     46 *          -7.8640   -1.1830   -0.7730 HGA2   46  UNL46       0.0000
     47 *          -7.8640   -0.9830    1.0230 HGA2   47  UNL47       0.0000
     48 *          -9.1200    1.1690    0.7850 HGA3   48  UNL48       0.0000
     49 *         -10.0040   -0.2100    0.0280 HGA3   49  UNL49       0.0000
     50 *          -9.1200    0.9690   -1.0150 HGA3   50  UNL50       0.0000
     51 *           9.9976    0.7558   -0.2181 BRGA1  51  UNL51       0.0000

**Reply by asukaorr** [reply]
Thanks @Jacqueline-teixeira.

One thing I noticed is that you have the Br- counter ion in the mol2 file. When using the CGenFF web server, you'll need to clean it out.

From the CGenFF program output, we can see that the charge and parameter penalty scores are 0 (see "param penalty=" and "charge penalty=" in the CGenFF program output, surfactant.str). So your molecule is well described in CGenFF.

[surfactant.zip](https://github.com/Sulstice/global-chem/files/9245839/surfactant.zip)

**Reply by Sulstice** [reply]
@asukaorr We did pretty well for a surfactant. I wonder what our tolerance is for this type of functional group styles.
